doc/quickstart/configure.rst

```diff
@@ -269,6 +269,38 @@ For example, Python's ``None`` is YAML's ``null``, Python's ``True`` is YAML's
    will be downloaded; otherwise, local data will be used.
 
 
+.. _config-logging:
+
+Logging configuration
+=====================
+
+These options can be specified under the ``"logging"`` section in the
+configuration file.
+
+Example:
+
+.. code:: yaml
+
+  logging:
+    log_progress_interval: 10s
```

Comment:
Maybe we should also mention how this can be used in the command line? E.g.

```bash
--logging='{"log_progress_interval": 2.0}'
```

This should go into the corresponding "Command line arguments" section further up.